app/build.gradle

```diff
@@ -98,6 +98,7 @@ android {
         }
     }
     lintOptions {
+        disable 'ObsoleteLintCustomCheck'
```

Comment:
Needed otherwise lint fails because the internal lint registry in the fragments dependency.